app/containers/ItunesContainer/TrackDetails/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ *
+ * TrackDetails
+ *
+ */
+
+import React, { useEffect, memo } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { useParams } from 'react-router';
+import { Skeleton } from 'antd';
+import SongCard from '@app/components/SongCard/index';
+import styled from 'styled-components';
+import T from '@app/components/T/index';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { selectItunesContainer, selectTrackDetails, selectTrackSearchError } from '../selectors';
+import itunesContainerSaga from '../saga';
+import { itunesContainerCreators } from '../reducer';
+import { Link } from 'react-router-dom';
+import { colors } from '@app/themes';
+import If from '@components/If/index';
+import { isEmpty } from 'lodash';
```

Comment:
Also nitpick. Structure your imports better. 
// third party
// alias
// relative

Will make it easier to read it I guess

Reply:
Updated